scipy/linalg/tests/test_lapack.py

```diff
@@ -1793,6 +1793,188 @@ def test_getc2_gesc2():
                                       x/scale, decimal=4)
 
 
+@pytest.mark.parametrize('size', [(6, 5), (5, 5)])
+@pytest.mark.parametrize('dtype', DTYPES)
+@pytest.mark.parametrize('joba', ['C', 'E', 'F', 'G', 'A', 'R'])
+@pytest.mark.parametrize('jobu', ['U', 'F', 'W', 'N'])
+@pytest.mark.parametrize('jobv', ['V', 'J', 'W', 'N'])
+@pytest.mark.parametrize('jobr', ['R', 'N'])
+@pytest.mark.parametrize('jobt', ['T', 'N'])
+@pytest.mark.parametrize('jobp', ['P', 'N'])
+def test_gejsv_general(size, dtype, joba, jobu, jobv, jobr, jobt, jobp):
```

Comment:
This currently tests *all* combinations of `job?`. This goes against our previous discussion, the reason I chose to add all is that whenever I thought I had considered all the edge cases I had slightly different case show up.  At least this way I can be confident that we haven't missed one.

This does add far too many test, and I will be happy to remove them (or mark them as xslow) but I'd us to be happy with the changes first.

Reply:
You're right, Kai. I think I was unclear and Sam changed to this, but I've asked him to change back above.

Reply:
👍 I'll keep them for now, just because I want to be confident that I haven't missed any edge cases.